### PR TITLE
APPS-1428 Update ofh azure region

### PR DESCRIPTION
**Pull request by Gvaihir**
(feat) Developing.md - updated docs
(feat) elsewhere - added new region

### CODE REVIEW

File: scripts/dxcompiler_copy/dxapp.json [hunk 1]
```diff
@@ -99,10 +99,17 @@
...
+      "systemRequirements": {
+        "*": {
+          "instanceType": "azure:mem2_ssd1_x2"
+        }
+      }
     }
   },
   "details": {
-    "whatsNew": "* 0.0.2: Added new London region."
+    "whatsNew": "* 0.0.3: Added new London OFH region * 0.0.2: Added new London region."
```
Comment (commandlinegirl) [inline scripts/dxcompiler_copy/dxapp.json]:
```suggestion
    "whatsNew": "* 0.0.3: Added new London OFH region\n* 0.0.2: Added new London region."
```

File: scripts/multi_region_tests.py [hunk 2]
```diff
@@ -25,7 +25,8 @@
             "dxCompiler_Sydney",
             "dxCompiler_Azure",
             "dxCompiler_Amsterdam",
-            "dxCompiler_Berlin"]
+            "dxCompiler_Berlin",
+            "dxCompiler_OFH_TRE"]
```
Comment (commandlinegirl) [inline scripts/multi_region_tests.py]: Hmm.. It looks like dxCompiler_London and dxCompiler_Europe_London are missing from here.

File: scripts/build_release.py [hunk 3]
```diff
@@ -37,7 +37,8 @@
     "azure:westeurope" : "dxCompiler_Amsterdam",
     "aws:eu-central-1" : "dxCompiler_Berlin",
     "aws:eu-west-2": "dxCompiler_London",
-    "aws:eu-west-2-g": "dxCompiler_Europe_London"
+    "aws:eu-west-2-g": "dxCompiler_Europe_London",
+    "azure:uksouth-ofh": "dxCompiler_OFH_TRE"
```
Comment (commandlinegirl) [inline scripts/build_release.py]: How about we name it dxCompiler_OFH_TRE_London (in case there will be a new region in the future for OFH-TRE)?

**Review by commandlinegirl — CHANGES_REQUESTED** [review]
Mostly minor requests. Thank you!

File: scripts/multi_region_tests.py (hunk 2 above)
Comment (Gvaihir) [reply scripts/multi_region_tests.py]: yes, I mentioned that in the comments to APPS-1428 - was not sure if that's intended or just an oversight. I'll add them

File: scripts/build_release.py (hunk 3 above)
Comment (Gvaihir) [reply scripts/build_release.py]: Sure

Comment (Gvaihir) [reply scripts/build_release.py]: fixed in [dd4730a](https://github.com/dnanexus/dxCompiler/pull/407/commits/dd4730a369b0c731832d98415aa7a5be7e02e0b2)

File: scripts/multi_region_tests.py (hunk 2 above)
Comment (Gvaihir) [reply scripts/multi_region_tests.py]: fixed in [dd4730a](https://github.com/dnanexus/dxCompiler/pull/407/commits/dd4730a369b0c731832d98415aa7a5be7e02e0b2)

File: scripts/dxcompiler_copy/dxapp.json (hunk 1 above)
Comment (Gvaihir) [reply scripts/dxcompiler_copy/dxapp.json]: fixed in [dd4730a](https://github.com/dnanexus/dxCompiler/pull/407/commits/dd4730a369b0c731832d98415aa7a5be7e02e0b2)

**Review by commandlinegirl — APPROVED** [review]
LGTM, thank you!

**Pull request merged**